util/cteutil/storage_test.go

```diff
@@ -14,78 +14,70 @@
 package cteutil
 
 import (
+	"github.com/stretchr/testify/require"
 	"reflect"
 	"strconv"
 	"testing"
 
-	"github.com/pingcap/check"
 	"github.com/pingcap/parser/mysql"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/chunk"
 )
 
-func TestT(t *testing.T) {
-	check.TestingT(t)
-}
-
-var _ = check.Suite(&StorageRCTestSuite{})
-
-type StorageRCTestSuite struct{}
-
-func (test *StorageRCTestSuite) TestStorageBasic(c *check.C) {
+func TestStorageBasic(t *testing.T) {
 	fields := []*types.FieldType{types.NewFieldType(mysql.TypeLong)}
 	chkSize := 1
 	storage := NewStorageRowContainer(fields, chkSize)
-	c.Assert(storage, check.NotNil)
+	require.NotNil(t, storage)
 
 	// Close before open.
 	err := storage.DerefAndClose()
-	c.Assert(err, check.NotNil)
+	require.NotNil(t, err)
```

Comment:
You may try to use `requuire.NoError` as a more precious checker.

Reply:
This test used check storage can not close before open ,  I think use require.Error may be better

Reply:
Oops, you're right. I always make mistake on boolean logic 😆